src/PresentationalComponents/TagsToolbar/ManageTags.js

```diff
@@ -56,13 +56,18 @@ const ManageTags = ({ toggleModal, fetchTags, selectedTags, setSelectedTags, isO
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [perPage, page]);
 
-    useEffect(() => {
-        setSelected(selectedTags.length ? selectedTags.map(id => ({
+    const populateSelected = () => {
```

Comment:
Just the one change, this function can be simplified to an expression.... but we'll merge anyways, this can be cleaned up at a later date